bot.py

```diff
@@ -108,6 +114,7 @@ def display_leaderboards(self, args: str, channel_id: str):
             userKarma = curKarma[curKarma['name'].str.startswith('<@')]
             thingKarma = pd.concat([curKarma,userKarma]).drop_duplicates(keep=False)
         except ValueError: # Empty file or no file present
+            self.logger.exception('Empty karma file or no file present, return.')
```

Comment:
This log should have some sort of remedy for this situation.

Reply:
Such as?  Really, this shouldn't happen anymore since we create the file at boot (changes in a different branch), right?

Reply:
Yeah, that's true. Maybe we can add some more stuff later for handling missing/empty files or something and provide more feedback? Out of scope for this change :+1: